### DISCUSSION TITLE
Why the behaviors of <c-f>, <c-b>, <c-d> and <c-u> are select rather scroll?

### DISCUSSION THREAD

**Discussion by leira** [71/dance#223] [General]
I'm a bit surprised when I first hit <c-f> in Dance. I was expecting scrolling a page, but it selected instead. Then I found <c-b>, <c-d> and <c-u> have the similar behavior, rather than scrolling. I double-checked the kakoune docs, they are mapped to scroll by default. What is the rational to change default behaviors from scroll in kakoune to select in Dance?

I changed the behavior to scroll using the setting below. I'm just curious on the reasons.

```json
  {
    "key": "ctrl+f",
    "command": "dance.select.vertically",
    "args": {
      "direction": 1,
      "by": "page",
      "shift": "jump"
    },
    "when": "editorTextFocus && dance.mode == 'normal'"
  },
```

**Comment by 71** [comment]
Looks like this was an error introduced by the switch to Dance 0.5 (2d38438e15724f13cb2896f39b56edbaa6eb97ac). See #224 for tracking.

**Comment by leira** [comment]
Thanks for the reply, @71. I see. Pre-#224, c-f was bound to scroll a page. Thanks for the quick fix!

https://github.com/71/dance/blob/1b3e222a3a0176717dfb963ff0bd486bf8e9defa/commands/commands.yaml#L106
```yaml
down.page:
  title: Scroll one page down
  keys: c-f (normal), c-f (insert)
```

https://github.com/71/dance/blob/1b3e222a3a0176717dfb963ff0bd486bf8e9defa/src/commands/move.ts#L120
```typescript
registerCommand(Command.downPage, CommandFlags.ChangeSelections, (_, { repetitions }) =>
  scrollBy(repetitions, "down", "page"),
);
```
